irods_completion.bash

```diff
@@ -2,12 +2,15 @@
 # Script for irods i-commands auto-completion with bash
 # This script is GPL, blah blah blah...
 # Bruno Bzeznik <Bruno.Bzeznik@imag.fr> 10/2011
-#  
+#
 # Simply source this script as follows:
-#     . irods_completion.bash 
+#     . irods_completion.bash
 # and enjoy <tab> key
 # Feel free to improve!
 #
+# Updated 2022/09/16 Terrell Russell
+#  - fix setting and unsetting IRODS_CWD (for 4.2+)
+#
 
 # Irods command to auto-complete
 command_list="ibun icd ichksum ichmod icp iget ils imeta imkdir imv iphybun iphymv irm irmtrash irsync itrim iput"
```

Comment:
Is this the list that is known to work or the actual complete set of supported operations? I wonder if `itouch`, `istream`, and friends can get in on this action?

Also, alphabetization would be nice :)

Reply:
yeah, didn't touch this...

correct, this is the list of commands that autocompletion reaches out and runs an ils (and parses its output(!)) for.

itouch and others could be added...

Reply:
Okay. I think this is fine for now. Resolving